playground/static-mpa/vite.config.ts

```diff
@@ -12,19 +12,11 @@ export default defineConfig({
 						contact: path.resolve(__dirname, 'contact.html'),
 						'404': path.resolve(__dirname, '404.html'),
 						about: path.resolve(__dirname, 'about/index.html'),
-						about404: path.resolve(__dirname, 'about/404.html'),
+						'about-404': path.resolve(__dirname, 'about/404.html'),
```

Comment:
why this change?

Reply:
Completely inconsequential. I just did it because I didn't like how it looked 😂.